caledonia/utils/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "utils"
```

Comment:
Instead of having another package, a `utils.rs` module in the other package would simplify the whole strucure?

Reply:
I first had a utils module but changed for some reasons... I will revert it back!